app/src/main/java/com/monkeyteam/chimpagne/model/database/AtomicChimpagneEventManager.kt

```diff
@@ -112,4 +112,45 @@ class AtomicChimpagneEventManager(
         .addOnSuccessListener { onSuccess() }
         .addOnFailureListener(onFailure)
   }
+
+  fun createPoll(
```

Comment:
Note that if you pass an already existing poll, the old one would be overwritten. I personally prefer to name the method "updatePoll". However this name is also not perfect, as you might think a poll needs to exist to be updated.

Reply:
Yes, but the intent of the function is to create a poll, and you cannot edit / update a poll once it has been created thus the situation you describe should never happen and is not the intended use case of this function.